common/ta_errors.h

```diff
@@ -179,6 +180,22 @@ typedef enum {
   SC_UTILS_TIMER_ERROR = 0x02 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
   /**< Errors occurred in timer function */
   SC_UTILS_TIMER_EXPIRED = 0x03 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< Failed to send message */
+  SC_UTILS_HTTPS_SEND_ERROR = 0x04 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS module initialize error */
+  SC_UTILS_HTTPS_INIT_ERROR = 0x05 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS X509 certificate parse error */
+  SC_UTILS_HTTPS_X509_ERROR = 0x06 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS initial connection error */
+  SC_UTILS_HTTPS_CONN_ERROR = 0x07 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS setting SSL config error */
+  SC_UTILS_HTTPS_SSL_ERROR = 0x08 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< Out of memory error */
+  SC_UTILS_OOM_ERROR = 0x09 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< Overflow error */
+  SC_UTILS_OVERFLOW_ERROR = 0x0A | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< Some other fault */
+  SC_UTILS_FAULT = 0x0D | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
```

Comment:
`SC_UTILS_FAULT` is hard to understand. Can you split into something meaningful?